Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /usr/src/app
 
 COPY . /usr/src/app
 
-RUN npm install --only=production
+RUN apk add git && npm install --only=production
```

Comment:
huh?

Reply:
It needed git to install your slackdown fork :man_shrugging:

Reply:
Stealing this for develop